matrix_sdk_base/src/client.rs

```diff
@@ -529,6 +527,19 @@ impl BaseClient {
             }
         }
 
+        // TODO:
+        // We wait until after the event has been decrypted?
```

Comment:
Yeah I was just making sure that we wanted the events decrypted I figured since the Store can be encrypted that this is Ok but I just wanted to make sure.